pywbem/_subscription_manager.py

```diff
@@ -541,6 +557,22 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True):
             permanent. See :ref:`WBEMSubscriptionManager` for details about
             these ownership types.
 
+          persistence_type (:class:`py:int` or None):
+            The PersistenceType property defines for the WBEM server
+            persistence of the destination instance as follows: 1-Other, 2-
+            Permanent, 3-Transient. Most WBEM servers set the value to
```

Comment:
If 1-Other is not allowed, then it should not appear in this enumeration. We should explain what this parameter supports, not what the CIM schema would allow. We can still have the sentence below that explains why Other is not supported.

Reply:
I suggest to change the format of showing the values from "2-Permanent" to "2 (Permanent)".

Reply:
Fixed  DONE